tools/release-scripts/release.js

```diff
@@ -50,37 +56,23 @@ if (argv.dryRun) {
 let version;
 
 const versionBumpOptions = {
-  preid: argv.preid,
+  preid: argv.docs ? 'docs' : argv.preid,
   type: argv._[0]
 };
 
-if (versionBumpOptions.type === undefined && versionBumpOptions.preid === undefined) {
-  console.log('Must provide either a version bump type, preid, or both'.red);
+if (!argv.docs && versionBumpOptions.type === undefined && versionBumpOptions.preid === undefined) {
+  console.log('Must provide either --doc or a version bump type, preid (or both)'.red);
   console.log(yargsConf.help());
   process.exit(1);
 }
 
-preConditions()
-  .then(test)
-  .then(versionBump(versionBumpOptions))
-  .then(v => { version = v; })
-  .then(() => addChangelog(version))
-  .then(() => {
-    return build(argv.verbose)
-      .catch(err => {
-        console.log('Build failed, reverting version bump'.red);
-
-        return exec('git reset HEAD .')
-          .then(() => exec('git checkout package.json'))
-          .then(() => exec('git checkout CHANGELOG.md'))
```

Comment:
I don't cut it out.

I've done refactoring and it is not just generated before build, so it doesn't need to cut out if build fails.